lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -376,7 +395,7 @@ def assign(src, dst)
     end
 
     def assign_later(parent:, child:)
-      buffers[:assignments] << [parent, child]
+      buffers[:assignments].merge!([parent.unique_identifier, child.unique_identifier] => [parent, child])
```

Comment:
Are we using `unique_identifier` for assignments and `id` for links? Whoops, my bad.